### PR TITLE
Searching tweets a vendor is mentioned by another user

### PR DESCRIPTION
**Pull request by misraya**
A request example:
http://localhost:3000/filter_user_tweets?user=ARealityEvent&mentioned=Apple&count=10

Let me know if anything is missing.

### CODE REVIEW

File: practice-app/back-end/routes/filter_user_tweets.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    paramq = 'from:' + username + ' @' + vendor;
+
+    //paramater to be used in get method
+    var params = {
+        q: paramq,
+        count: req.query.count
+    }
+
+    //initialize empty list to collect data
+    tweetList = [];
```
Comment (kayacanv) [inline practice-app/back-end/routes/filter_user_tweets.js:24]: it should be better to initialize, where you used (inside the brackets in the bellow)

File: practice-app/back-end/routes/filter_user_tweets.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+            //add text parts of tweets to the list
+            for (var i = 0; i<tweets.length; i++)
+                tweetList.push(tweets[i].text)
+
+            //convert to json    
+            var res_str = JSON.stringify(tweetList);
+
+            //send JSON response
+            res.json(res_str);
```
Comment (kayacanv) [inline practice-app/back-end/routes/filter_user_tweets.js:41]: one line is better I think, your choice  res.json(JSON.stringifyt.....);

File: practice-app/back-end/routes/filter_user_tweets.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+    //initiliaze twitter object with token from key.js
+    var Twitter = require('twitter');
+    var key = require('./keys.js');
+    var client = new Twitter(key);
+
+    //convert request parameters to get parameters
+    username = req.query.user;
+    vendor = req.query.mentioned;
+    paramq = 'from:' + username + ' @' + vendor;
```
Comment (kayacanv) [inline practice-app/back-end/routes/filter_user_tweets.js:15]: generally declaring new variables is not suggested thing,  still it is okay

**Comment by misraya** [comment]
Updated accordingly.

**Pull request merged**